src/main/java/com/dotloop/loopit/LoopItController.java

```diff
@@ -54,51 +51,102 @@
     @Value("${dotloop.api.endpoint}")
     private String apiEndpoint;
 
-    @RequestMapping("/loopit")
+    private String username = "api user";
+
+    private Request attachHeaders(Request request) {
+        Token token = TokenStore.get(username);
+        request.addHeader("Authorization", "Bearer " + token.getAccessToken())
+                .addHeader("Content-type", "application/json");
```

Comment:
@sweetcoco prob makes sense to use `ContentType.APPLICATION_JSON` constant here as well